src/Runner.Worker/StepsRunner.cs

```diff
@@ -92,26 +92,11 @@ public async Task RunAsync(IExecutionContext jobContext)
                 var envContext = new CaseSensitiveDictionaryContextData();
 #endif
 
-                // Global env
                 foreach (var pair in step.ExecutionContext.EnvironmentVariables)
```

Comment:
These changes are to revert the workaround we used to flow Env to composite actions. This is now handled in the Composite Steps Runner.